pkg/network/netlink/conntracker.go

```diff
@@ -42,10 +45,22 @@ type connKey struct {
 	transport network.ConnectionType
 }
 
+type translationEntry struct {
+	*network.IPTranslation
+	orphan *list.Element
+}
+
+type orphanEntry struct {
+	key connKey
+	ttl time.Time
+}
+
 type realConntracker struct {
 	sync.RWMutex
-	consumer *Consumer
-	state    map[connKey]*network.IPTranslation
+	consumer      *Consumer
+	cache         *simplelru.LRU
```

Comment:
I wonder if this might be easier to comprehend if the LRU and orphan list were abstracted into a separate type from Conntracker. It would isolate the Conntracker from a lot of the orphan logic since that seems to apply when entries are add/removed/read.

Reply:
Done, see https://github.com/DataDog/datadog-agent/pull/7677/commits/7dc18290db323e87bc7f27c1f9b429aa83a35b8f